lib/utils.js

```diff
@@ -134,40 +134,91 @@ module.exports = {
   },
 
   /**
+   *
+   * References:
+   * - https://github.com/facebook/react-native/blob/main/Libraries/Utilities/Platform.ios.js
+   * - https://github.com/facebook/react-native/blob/main/Libraries/Utilities/NativePlatformConstantsAndroid.js
+   *
    * @returns An object with names and versions of the various components making up the context.
    */
   getVersions() {
     const packageJson = require("../package.json");
-    const packageVersion = packageJson.version;
+    const sdkVersion = packageJson.version;
     const environment = this.getEnvironment();
 
+    const sdk = "JS";
+
     try {
       if (environment === "reactnative") {
         const { Platform } = require("react-native");
+        let deviceName = "unknown";
+        let deviceVersion = "unknown";
+        if (Platform.OS === "ios") {
+          if (Platform.isPad()) {
+            deviceName = "iPad";
+          } else if (Platform.isTV()) {
+            deviceName = "AppleTV";
+          } else {
+            // RN doesn't support Apple Watch
+            deviceName = "iPhone";
+          }
+          deviceVersion = "unknown";
+        } else if (Platform.OS === "android") {
+          deviceName = `${Platform.Manufacturer}:${Platform.Brand}`;
+          deviceVersion = `${Platform.Model}`;
```

Comment:
The [react native docs](https://reactnative.dev/docs/platform#constants) on this, describes the `Model` as:
> The end-user-visible name for the Android device.

What is the intended semantics of this property exactly?

Reply:
The specification split "Samsung GT-I9100" so deviceName is "Samsung" and deviceVersion is "GT-I9100", and I have interpreted deviceVersion as the model.